lib/AST/ASTVerifier.cpp

```diff
@@ -2715,6 +2715,37 @@ class Verifier : public ASTWalker {
       verifyCheckedBase(nominal);
     }
 
+    void verifyChecked(GenericTypeParamDecl *GTPD) {
```

Comment:
Are any of these properties things we can meaningfully check in verifyParsed or verifyCheckedAlways (i.e. "verifyCheckedEvenIfThereAreErrorsAfterParsing")?

Reply:
@slavapestov Question still relevant!

Reply:
verifyCheckedAlways() should be OK, but not verifyParsed() since the depth isn't set yet.